docs/general/passing_arbitrary_options.md

```diff
@@ -0,0 +1,45 @@
+[Back to Guides](../README.md)
+
+# Passing Arbitrary Options To A Serializer
+
+Let's say you have a basic Post Controller:
+
+```ruby
+class PostController < ApplicationController
+  def dashboard  
+    render json: @posts
+  end
+end
+```
+
+Odds are, your serializer will look something like this:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
+end
+```
+
+This works all fine and well, but maybe you passing in some arbitrary options
+into the serializer. These options can be anything that isn't already reserved for use by
+ActiveModelSerializers' adapter options.
```

Comment:
Perhaps we can simplify the above to

```plain
# Passing Arbitrary Options To A Serializer

In addition to the [`serialization_scope`](serializers.md#scope), any options passed to `render`
that are not reserved for the [adapter](rendering.md#adapter_opts)
are available in the serializer as [instance_options](serializers.md#instance_options).

For example, we could pass a `user_id` option:

     ```ruby
    # posts_controller.rb
    etc
```

Docs for serialization_scope are stuck in https://github.com/rails-api/active_model_serializers/pull/1252